config.yaml.example

```diff
@@ -38,6 +38,16 @@ source:
   fetchSize: 1000
   # Optional condition to filter source table data that will be migrated
   # where: race_start_date = '2015-05-27' AND race_end_date = '2015-05-27'
+  # The following two configs are only applied to discord.messages table data migrations.
+  # Optional bucket value to skip copying data in discord.messages table from cassandra to scylla
+  # after the given bucket.
+  # messages_table_cutover_bucket: 270
+  # Optional max screw time value in days for discord.messages table data migrations.
+  # preserveTimestamps must be false because discord.messages table has collections type.
+  # For messages before cutover bucket, we will pick a write timestamp that is before we read the chunk
+  # from the token range and use that timestamp on the row written to scylla.
+  # Default: 10 days.
+  # messages_table_max_screw_time_in_days: 10
```

Comment:
I think this makes sense based on @jhgg's algorithm. But I'm also wondering if works if we pick just an absolute time from the past? Need to think through that part. But at least we know we can hack a writetime for these.